### DISCUSSION TITLE
Spike Vue 3 / Nuxt.js

### DISCUSSION THREAD

**Discussion by davidmyersdev** [davidmyersdev/octo#117] [Ideas]
I have been watching Vue 3 (which was officially released today), and I also have been looking into Nuxt.js. Based on the current level of complexity of this app, I think Nuxt could be a good fit. I'd like to experiment with it, and if Nuxt is ready to support Vue 3, this could help tackle both at once.

**Comment by davay42** [comment]
Vue 3 is great! It's faster, more flexible and future proof. I love that still it is very usable as a standalone script right in the browser, so it's compatible with the predictions about all the bundling and building becoming obsolete. But for now Nuxt looks very powerful indeed )
